tests/test_configuration_set.py

```diff
@@ -101,3 +104,96 @@ def test_configurations_load_xyz():
     for config in configs:
         assert config.charge == 0
         assert config.mult == 2
+
+
+@work_in_tmp_dir()
+def test_configurations_load_xyz_with_energies_forces():
+    """
+    Test loading an xyz file for a config set wth variable energies,
+    forces and box sizes.
+    """
+    configs = ConfigurationSet()
```

Comment:
Check if it is possible to define this as a pytest fixture. If yes, it could be reused by all three tests.